security/lynis/Portfile

```diff
@@ -38,22 +39,43 @@ build {}
 set target_dir ${prefix}/etc/${name}
 
 pre-destroot {
-    reinplace -E "s|(t.+_TARGETS=)\".+\"|\\1\"${target_dir}\"|g" \
+    reinplace -E "s|(t.+_TARGETS=.+) \.(.*)\"|\\1 .\\2 ${target_dir}\\2\"|g" \
         ${worksrcpath}/lynis \
         ${worksrcpath}/include/functions
 }
 
 destroot {
+    # Main binary
     xinstall -m 755 ${worksrcpath}/${name} ${destroot}${prefix}/bin/${name}
+
+    # Additional config
     xinstall -d ${destroot}${target_dir}
     file copy ${worksrcpath}/db \
               ${worksrcpath}/include \
               ${worksrcpath}/plugins \
               {*}[glob ${worksrcpath}/*.prf] \
         ${destroot}${target_dir}
+
+    # Man page
     xinstall ${worksrcpath}/${name}.8 ${destroot}${prefix}/share/man/man8
+
+    # Bash completion
+    set completions_path ${prefix}/share/bash-completion/completions
+    xinstall -d ${destroot}${completions_path}
+    xinstall -m 644 ${worksrcpath}/extras/bash_completion.d/${name} \
+        ${destroot}${completions_path}
 }
 
 livecheck.type      regex
 livecheck.url       ${master_sites}
 livecheck.regex     >${name}-(\\d+(\\.\\d+)+)${extract.suffix}<
+
+notes "
+You are urged not to store custom profiles and plugins within the
+${prefix}/etc/${name} directory as this content is wiped during
+upgrades and uninstallation.
+
+Instead, use any of the directories that lynis searches within by
+default, or use the command line options to specify custom paths.
+See `man lynis`.
+"
```

Comment:
This is perhaps an argument for putting this stuff in lib or what have you and setting ${prefix}/etc/${name} as the place where one puts the config...

Reply:
I did consider that argument, but I thought this solution was simpler and minimises changes to upstream behaviour.

Reply:
Let me know what you think, @pmetzger. I still believe what I've committed is the cleanest solution here.

Reply:
The issue is, we end up blowing away things in the configuration directory without the user expecting it. That's probably pretty bad. Might want an opinion from someone like @ryandesign.

Reply:
It seems I misunderstood the documentation here. After doing some testing, only the destrooted files get wiped, and so any custom plugins or profiles will be persisted.

Since the destrooted files are just defaults and templates, the user should not be editing these, but instead, should be creating new files for custom plugins and profiles, which will be persisted. Therefore in my latest commit, I've removed the warning, and gone back to only looking for config files in the `${prefix}/etc/lynis/*` directories.